src/pages/utils.js

```diff
@@ -0,0 +1,6 @@
+export function convertUnits(minUnit, maxUnit) {
+    if(minUnit === 0) {
```

Comment:
One problem with this is that there are classes that actually do offer a 0 credit option. I mean I guess it would be fine since a majority of classes may not apply to that

Reply:
For classes do offer 0 credit, the maxUnit will also be 0 so it won't affect that in that area. Correct me if I'm wrong but even for courses with options to choose credits, the minimum credit is usually 1. So we can rule out option of 0 credit. The courses that do offer 0 credit are usually minUnit-maxUnit = 0-0

Reply:
I see what you are getting at, the only course that offers 0 or 1 credit that I know is my Steinhardt Choir Ensemble. Honestly, I think that this is rare enough that making this change should be fine. @EthanPrintz, what do you think?

Reply:
Ah, I see. I think an approach we can take is remove minUnit if it's 0 for now, then we add extra handling for exception courses that might be outliers, which I think rare enough to be hardcoded in